Formula/geogram.rb

```diff
@@ -0,0 +1,25 @@
+class Geogram < Formula
+  desc "Programming library of geometric algorithms"
+  homepage "http://alice.loria.fr/software/geogram/doc/html/index.html"
+  url "https://gforge.inria.fr/frs/download.php/file/37635/geogram_1.6.6.tar.gz"
+  sha256 "08211b1d6f21e14701e3fd5c79adbe331cdf66b8af84efdb54cd7048244691b5"
+  depends_on "cmake" => :build
+
+  def install
+    # don't write directly into /usr/local/lib/cmake/modules
+    inreplace "CMakeLists.txt", "lib/cmake/modules", "#{share}/cmake/Modules"
+    # workaround for undefined _mm_set_pd1 in clang
+    inreplace "src/lib/geogram/numerics/predicates.cpp", "_mm_set_pd1", "_mm_set1_pd"
+    open("CMakeOptions.txt", "a") do |f|
+      f << "\nset(CMAKE_INSTALL_PREFIX #{prefix})\n"
+    end
+    system "sh", "-f", "configure.sh"
+    Dir.chdir("build/Darwin-clang-dynamic-Release")
+    system "make", "install"
+  end
+
+  test do
+    system "curl", "https://raw.githubusercontent.com/FreeCAD/Examples/master/Point_cloud_ExampleFiles/PointCloud-Data_Stanford-Bunny.asc", "-o", "bunny.xyz"
```

Comment:
Use a `resource do` block in the main body and then stage this in `test do` that way. It'll need to be pinned to a commit though, not simply fetching `master`.

Reply:
I've made a change for this - I'm not sure if there's a more idiomatic way than `File.rename` inside a resource `stage` block to handle the URL filename differing from the needed filename (the test needs the `.xyz` extension instead of `.asc`).